android/library/src/main/java/com/liferay/mobile/screens/thingscreenlet/screens/BaseScreenlet.kt

```diff
@@ -0,0 +1,13 @@
+package com.liferay.mobile.screens.thingscreenlet.screens
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.FrameLayout
+
+open class BaseScreenlet @JvmOverloads constructor(
```

Comment:
Do we need this extracted?

Reply:
No, we can keep it inside ThingScreenlet file